### PR TITLE
Feature: return model from mintWZCN in wasmzcn

### LINKED ISSUE #636: Feature: return model from mintWZCN in wasmzcn

### PR DESCRIPTION
**Pull request by YarikRevich**
#Refers to issue #636 
#Depends on the issue #635

### CODE REVIEW

File: zcnbridge/bridge.go [hunk 1]
```diff
@@ -275,13 +275,16 @@ func (b *BridgeClient) BurnWZCN(ctx context.Context, amountTokens uint64) (*type
...
+	//clientID = []byte(hex.EncodeToString(clientID))
+	//fmt.Println(b.ClientID())
+	//return nil, nil
 
 	// 2. Data Parameter (signature)
 	amount := new(big.Int)
 	amount.SetInt64(int64(amountTokens))
 
-	bridgeInstance, transactOpts, err := b.prepareBridge(ctx, "burn", amount, clientID)
+	bridgeInstance, transactOpts, err := b.prepareBridge(ctx, "burn", amount, b.ClientID())
```
Comment (peterlimg) [inline zcnbridge/bridge.go]: Please use the same data type of ClientID for gas estimation and Brun. Seems the previous code was correct.

File: zcnbridge/bridge.go [hunk 2]
```diff
@@ -275,13 +275,16 @@ func (b *BridgeClient) BurnWZCN(ctx context.Context, amountTokens uint64) (*type
 	}
 
 	// 1. Data Parameter (amount to burn)
-	clientID := DefaultClientIDEncoder(b.ClientID())
+	//clientID := DefaultClientIDEncoder(b.ClientID())
+	//clientID = []byte(hex.EncodeToString(clientID))
+	//fmt.Println(b.ClientID())
+	//return nil, nil
```
Comment (peterlimg) [inline zcnbridge/bridge.go]: Remove unused code and adjust the comments below

**Review by peterlimg — DISMISSED** [review]

**Review by cnlangzi — APPROVED** [review]
LGTM

**Review by dabasov — APPROVED** [review]
lgtm

**Pull request merged**